### ISSUE TITLE
Sentence case vs Title Case

### ISSUE THREAD

**Issue by peterdesmet** [tdwg/website-jekyll#27]
@gkampmeier @stanblum What is the recommendation for titles on the TDWG website? Titles are page titles, page headings (h2, h3), and menu items.

- Title Case
- Sentence case

Currently a mix of the two is used. See https://medium.com/@jsaito/making-a-case-for-letter-case-19d09f653c98 for the benefits of each.

**Comment by gkampmeier** [comment]
@peterdesmet sentence case

**TL;DR**
Nice article--he does, however, screw up the initial definition of Title Case, which he demonstrates later. But in so doing, points out the perils of title case: what are the rules (there are many style guides)?  For our community, an international one, it likely makes more sense to use sentence case--mind you, before reading that article, I might have pushed back otherwise. The rules are simpler, as long as you can call out the text in an h-heading.  Although I seem to remember that different h-headings can have different rules (title for h1&2; sentence for h3&4)—but we don't need to go there.

**Comment by peterdesmet** [comment]
Thanks, I agree, it is indeed simpler.

TDWG **standard** names will remain capitalized:
```
Darwin Core
Global Genome Biodiversity Network
```

But what about the following terms:

```
executive committee
functional subcommittee
- fundraising and partnership
- time and place
- technical architecture group
- infrastructure
- outreach and communications
task group
interest group
maintenance group
code of conduct
diversity and inclusion statement
liability disclaimer and release
...
```

My preference would be that those are lowercase.

E.g. in a title or sentence:

```
TDWG members of the executive committee participated in the BiCIKL hackathon in Meise
The Audubon Core maintenance group has opened public comment on the proposal to add two vocabularies to the Audubon Core standard.
 Interest and task group working sessions
```

Or as sentence case menu items:

```
About
---
Executive committee
---
Functional subcommittees
- Fundraising and partnership
- Time and place
- Technical architecture group (TAG)
- Infrastructure
- Outreach and communications
---
Code of conduct
Diversity and inclusion statement
```

**Comment by gkampmeier** [comment]
@peterdesmet I agree that Standard names should remain capitalized.  I would suggest the following (I have not made things 3 deep, but could see advantages to this in some cases).  Still noodling here, but...

About
- Constitution
- Membership
- Incorporation and Status
- Contact
---
Executive Committee†
Functional Subcommittees
- Fundraising and Partnership
- Time and Place
- Technical Architecture Group (TAG)
- Infrastructure
- Outreach and Communications
---
Policies and Statements
- Terms of Use
- Code of Conduct
- Diversity and Inclusion
- Liability Disclaimer and Release - COVID-19
---
Process and Procedures
- TDWG Process
- Review Manager Guidelines
- Contributing to the TDWG website
- Logos††

†Technically, the Exec includes FSc (as well as Regional Reps, which might be encouraged to have more of a presence on the website). However, there are a bunch of historical background things currently as cards hanging off of the Exec page, which we want to be able to preserve under a history (likely its own new section).

†† should this be "Branding Guidelines" with the logos as resources?

**Comment by peterdesmet** [comment]
Thanks for the suggestion on the about menu.

1. I do notice you are using Title Case (rather than the suggested Sentence case) for that menu. Is that a result of copy/pasting or intended?
2. Can you indicate what we should recommend for the terms (executive committee, interest groups, etc.) I listed in https://github.com/tdwg/website-jekyll/issues/27#issuecomment-1327577363? **Can we use lowercase for all these?** That would at least avoid the current variations like [these](https://www.tdwg.org/news/2022/call-for-nominations/#functional%20subcommittee%20for%20outreach%20and%20communications%20chair):

> ### Functional **s**ubcommittee for **O**utreach and **c**ommunications **c**hair
> The motivation, mission and responsibilities of the **f**unctional **s**ubcommittee are explained on the **s**ubcommittee’s page. The **c**hair provides leaderships on **o**utreach & **c**ommunications for the whole of TDWG.
> 
> ### Functional **s**ubcommittee for **F**unding and **p**artnerships chair
> The motivation, mission and responsibilities of the **f**unctional **s**ubcommittee are explained on the **s**ubcommittee’s page. The **c**hair provides leaderships on **F**undraising and **P**artnerships for the whole of TDWG.
> 
> ### **R**egional **R**epresentatives: North America, Africa, Asia & Europe
> Regional **r**epresentatives give voice to the members, institutions, and projects in their region, and collectively provide a global perspective to the **E**xecutive **C**ommittee. As members of the **E**xecutive **C**ommittee, regional representatives have responsibility for the oversight of TDWG and the authority to approve/decline strategic initiatives, budgets, and all aspects of operations and activities. The **E**xecutive **C**ommittee also approves or declines the formation of **I**nterest and **T**ask **G**roups, as well as the ratification of standards and best practice specifications produced by **I**nterest and **T**ask **G**roups.

I would have all bolded letters above be lowercase.

**Comment by gkampmeier** [comment]
@peterdesmet yes it was intentional because these are proper names for groups and policies.  I don't think that these named entities should be lower case because we are referring to a particular entity/proper name (TDWG Executive Committee) not just any old exec.

These are different from other sorts of titles for headings, to me.  If I were to refer to the Executive Committee (meaning TDWG Exec) or the Program Committee (not the concept of a program committee), there is a difference. It is harder for the reader to know if you don't capitalize these.  The practice of putting everything [e e cummings](https://en.wikipedia.org/wiki/E._E._Cummings) is not a comfortable space for me.

**Comment by stanblum** [comment]
+1 for capitalizing proper nouns (names).  The problem I have is determining what is part of the name; e.g., 
- "Executive Committee" versus "Executive" committee;
- subcommittee, functional subcommittee (kind of subcommittee), so 
    - Functional subcommittee for Outreach and Communication,
    - Functional Subcommittee for Outreach and Communication?
- representative: 
    - "Regional representative for Africa" 
    - "Regional Representative for Africa"?
 
On Gail's menu items and grouping:

Overall, I like the grouping and order; would just suggest that "Contributing to website" and "Logos" aren't the same kind of Process and Procedures as the "TDWG Process" and "Review Manager Guidelines". Those are names of documents.  Also, if Steve Baskauf's suggestion is developed, "Procedures" might become a top-level menu item.

I don't think "Incorporation and Status" is the name of something, so leave status lowercase, right?

**Comment by peterdesmet** [comment]
I agree with Stan that we should define how to capitalize those groups (`Functional Subcommittee` vs `Functional subcommittee`, `Interest Group` vs `Interest group`). I think in most cases we have chosen to capitalize both, and we could stick to that, but **we should have a clear list**.

I would not capitalize roles and policies, as that can get unwieldy, so I prefer `deputy chair`, `convener`, `representative`, `code of conduct`. The context is almost always clear in sentences.

In the menu Gail suggests, I would thus write (capitalizion changes only):
> Incorporation and status
> Policies and statements
> Terms of use
> Code of conduct
> Diversity and inclusion
> Liability disclaimer and release - COVID-19
> Process and procedures
> TDWG process (arguable, since also IG?)
> Review manager guidelines

**Comment by gkampmeier** [comment]
@peterdesmet I am OK with the capitalization changes you list just above and the idea of having a clear list of how groups should be capitalized.  I think that if you refer to a specific person in a role, that this should be capitalized, however (e.g., Ely Wallis is Deputy Chair of TDWG; Holly Little is Representative for North America). Similarly, referring to TDWG's Code of Conduct, I would capitalize it because it is not just _a_ code of conduct but _our_ Code of Conduct.

You mention that "TDWG process" is also the title of an IG, but this document is an output from that IG.  You could change the wording to TDWG framework, but I think that is more opaque to the reader.
@stanblum This took me back to the [Process IG](https://www.tdwg.org/community/process/) webpage, which has broken links and a misuse of capitalization for Lee belbin[sic] ;)

**Comment by stanblum** [comment]
To my understanding, the "TDWG Process"  is a named thing, like our
Constitution, so Process should be capitalized.

**Comment by peterdesmet** [comment]
Note: my preference would be to not capitalize roles, as I find it rather unwieldy to have:

> Mareike Petersen, Chair of the Functional Subcommittee of Outreach and Communications

But I can adapt. 😄

Most of all, can you (@gkampmeier or @stanblum) provide a list of words that should be consistently capitalized? That should help me and others in reviewing/contributing content for the website.

**Comment by gkampmeier** [comment]
@peterdesmet @stanblum I've started a Google doc in the TDWG docs > Guidelines > [Capitalizations Guidelines - TDWG Website](https://docs.google.com/document/d/15_0uW1X6yhSPzKk6UC7upNhO0pE1_MXsRx4FGvV4lwY/edit). Suggestions for how it should be organized are welcomed.

**Comment by peterdesmet** [comment]
I have tried to stick to the rules in the document:

- [x] Executive Committee (also Exec)
- [x] Chair **only for exec**
- [x] Deputy Chair **only for exec**
- [x] Secretary
- [x] Treasurer
- [x] Functional Subcommittees **not consistent in Constitution**
- [x] Technical Architecture Group
- [x] Fundraising and Partnerships
- [x] Infrastructure
- [x] Outreach and Communications
- [x] Time and Place
- [x] Regional Representative for [Region]
- [x] Administrator, TDWG Secretariat
- [x] Editor-in-Chief
- [x] Program Committee (Chair, Co-Chair etc. when paired with a specific person)
- [x] Local Committee (Chair, Co-Chair etc. when paired with a specific person) **not encountered**
- [x] Named TDWG policies and TDWG standards should have major words capitalized; generic policy
- [x] Code of Conduct
- [x] Contributing to the TDWG Website (as title); contributing to the TDWG website (in a sentence) **Renamed `Contributing to the Website`
- [x] By-laws (formerly TDWG Process on About) **not consistent in constitution, called `Process` in menu, `TDWG Process` on the page**
- [x] Constitution
- [x] Incorporation and Status
- [x] Letter of Determination
- [x] Liability Disclaimer and Release - COVID-19
- [x] Logos (as title); logos (in a sentence)
- [ ] Membership (as policy); membership (in sentence) **not verified**
- [x] Review Manager guidelines **Written as `Review Manager Guidelines`**
- [x] Diversity and Inclusion statement **Written as `Diversity and Inclusion Statement`**
- [x] Terms of Use
- [x] Interest Group names **not verified, but typically title case**
- [x] Task Group names **not verified, but typically title case**
- [x] Standard names **not verified, but typically title case**
